### DISCUSSION TITLE
Add Tab size option for code preview

### DISCUSSION THREAD

**Discussion by IGR2014** [community/community#69831] [Mobile]
Tab size would be useful setting for better visual experience. In web version predefined tab size is 8 and some users like me are automatically using this settings for their projects. While in mobile app this size is 4 so some indentation looks broken.

P.S. If app is opensource I can take implementation on myself

**Comment by elisealix22** [comment]
Hi @IGR2014 , thanks for reaching out! Are you seeing this on Android or iOS? In the meantime, I'll forward this along to our team and we will do our best to resolve.

**Reply by IGR2014** [reply]
Sorry, forgot to mention. Experiencing this on Android 10 (Xiaomi Redmi Note 7)
